packages/services/src/competition.service.ts

```diff
@@ -1322,18 +1322,21 @@ export class CompetitionService {
    */
   private async calculatePendingCompetitionLeaderboard(
     competitionId: string,
+    type: CompetitionType,
   ): Promise<LeaderboardEntry[]> {
-    const agents =
+    const agentIds =
       await this.competitionRepo.getCompetitionAgents(competitionId);
-    const globalLeaderboard =
-      await this.agentScoreRepo.getAllAgentRanks(agents);
+    const globalLeaderboard = await this.agentScoreRepo.getAllAgentRanks({
```

Comment:
example of how the global rank is, technically, used when we calculate the pending comps order. this'll make it relative to the comp. 

e.g., if agent ABC is #1 in trading, but never participated in perps, they'll be treated like the rest of the "never participated" agents. we could get fancy here, but this is easier.